release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -354,6 +354,11 @@ For more information, see xref:../security/tls-security-profiles.adoc#tls-securi
 
 Previously, the `oauth-proxy` command only allowed the use of SHA-1 hashed passwords in `htpasswd` files used for authentication. `oauth-proxy` now includes support for `htpasswd` entries that use `bcrypt` password hashing. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1874322[BZ#1874322].
 
+[id="ocp-4-8-managed-secure-boot"]
+==== Enabling managed Secure Boot with installer-provisioned clusters
+
+{product-title} 4.8 supports automatically turning on UEFI Secure Boot mode for provisioned control plane and worker nodes and turning it back off when removing the nodes. To use this feature, set the node's `bootMode` configuration setting to `UEFISecureBoot` in the `install-config.yaml` file. Red Hat only supports installer-provisioned installation with managed Secure Boot on 10th generation HPE hardware and 13th generation Dell hardware running firmware version `2.75.75.75` or greater. For additional details, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-managed-secure-boot-in-the-install-config-file_ipi-install-configuration-files[Configuring managed Secure Boot in the install-config.yaml file].
```

Comment:
@johnwilkins I wonder if you should change the HPE hardware and 13th generation Dell hardware running firmware version `2.75.75.75` or greater to an or statement??
The first time I read it, I thought the firmware version 2.75.75.75 was required for both HPE and Dell. Which makes no sense, really, as they have their own firmware and versioning.